torchinfo/layer_info.py

```diff
@@ -10,6 +11,14 @@
 ]
 
 
+# using wonder's beautiful simplification: https://stackoverflow.com/a/31174427/8890157
+def rgetattr(obj: object, attr: str, *args) -> object:
```

Comment:
Add `: Any` to *args type annotation

Reply:
Also I think obj is always of type `str`

Reply:
`obj` would be the module itself whose submodule/attribute we are trying to extract. I am not sure if it would be a string.

Reply:
I rewrote this whole thing and the issue is now fixed